ingestion/src/metadata/ingestion/source/database/dbt_source.py

```diff
@@ -309,6 +312,30 @@ def process_dbt_lineage_and_descriptions(self) -> Iterable[AddLineageRequest]:
                         f"Failed to parse the node {upstream_node} to capture lineage: {exc}"
                     )
 
+            # Create Lineage from DBT Queries
+            try:
+                service_database_schema_table = fqn.split(data_model_name)
+                target_table_fqn = ".".join(service_database_schema_table[1:])
+                service_name = service_database_schema_table[0]
+                database_name = service_database_schema_table[1]
+                database_schema_name = service_database_schema_table[2]
+                create_statement = f"create table {target_table_fqn} as"
+                query = f"{create_statement} {data_model.sql.__root__}"
+                lineages = get_lineage_by_query(
```

Comment:
You might be able to get rid of the `too-many-locals` linting by moving these assignments directly inside the `get_lineage_by_query` function. These variables are just being used once inside the function, so having them aside is not really adding much value.

Another possibility would be to split the function in smaller pieces, but it might not be needed in this case.

Thanks!

Reply:
Got it will do that 👍